core/src/main/scala/org/ensime/indexer/lucene/SimpleLucene.scala

```diff
@@ -16,12 +17,22 @@ import org.apache.lucene.util._
 
 import scala.collection.JavaConverters._
 import scala.collection.mutable
+import scala.concurrent.{ ExecutionContext, Future, blocking }
 
 object SimpleLucene {
   // from DirectDocValuesFormat.MAX_SORTED_SET_ORDS = 2147483391 but
   // it's kind of an insane number, so let's pick something halfway
   // sensible.
   val MaxResults = 10000
+
+  // our fallback analyzer
+  class LowercaseAnalyzer extends Analyzer {
```

Comment:
It has no dependencies on any members or types defined in the class, so made more sense to me to make it an independent helper class. WDYT?